src/main/java/seedu/address/storage/JsonAdaptedClient.java

```diff
@@ -162,8 +182,13 @@ public Client toModelType() throws IllegalValueException {
         }
 
         final Set<Tag> modelTags = new HashSet<>(clientTags);
-        return new Client(modelName, modelPhone, modelEmail, modelAddress, modelCountry,
-            modelTimezone, modelContractExpiryContractExpiryDate, modelLastModifiedInstant, modelTags);
+        Client modelClient =
+                new Client(modelName, modelPhone, modelEmail, modelAddress, modelCountry, modelTimezone,
+                    modelContractExpiryContractExpiryDate, modelLastModifiedInstant, modelTags);
```

Comment:
formatting, breaking a line up after `=` makes it inconsistent with the rest of the code base